### DISCUSSION TITLE
Value configuration for specific device

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js-ui#2146] [General]
I think the answer to this is "no", but thought I would ask anyway.

Is there a way to offset values on a specific device (not all devices of the same type)? For instance, add/subtract an offset value from a humidity or temperature sensor to calibrate it to a known value?

**Comment by robertsLando** [comment]
@Botched1 If you are using zwavejs2mqtt with mqtt gateway you could do this using gateway values table with a custom send function

**Reply by Botched1** [reply]
I hadn't thought of that. Thanks for the tip!
